### DISCUSSION TITLE
Code inside PXR namespace

### DISCUSSION THREAD

**Discussion by rlei-weta** [Autodesk/maya-usd#877] [Q&A] [answered]
Hello,

I've noticed a good portion of the code are inside the PXR namespace - what is the long term plan of where the symbols will reside?

Thanks!
Richard

**Comment by kxl-adsk** [comment, marked as answer]
The fact that `maya-usd` repository has many origins cannot be hidden and it will take time to get some of it cleaned up. I wouldn't expect anything to change in the short term unless there are clever ways of refactoring this code (I have some hopes with clang-rename, but haven't tried it yet).

**Reply by rlei-weta** [reply]
Thanks for your response @kxl-adsk - `clang-rename` does look promising!
